src/components/output/index.tsx

```diff
@@ -20,26 +21,29 @@ export default class App extends Component<Props, State> {
   pinchZoomRight?: PinchZoom;
   retargetedEvents = new WeakSet<Event>();
 
-  updateCanvases(img: ImageBitmap) {
-    for (const [i, canvas] of [this.canvasLeft, this.canvasRight].entries()) {
-      if (!canvas) throw Error('Missing canvas');
-      const ctx = canvas.getContext('2d');
-      if (!ctx) throw Error('Expected 2d canvas context');
-      if (i === 1) {
-        // This is temporary, to show the images are different
-        ctx.filter = 'hue-rotate(180deg)';
-      }
-      ctx.clearRect(0, 0, canvas.width, canvas.height);
-      ctx.drawImage(img, 0, 0);
-    }
+  updateCanvas(canvas: HTMLCanvasElement, img?: ImageBitmap) {
```

Comment:
Doesn't look like this needs to be a member of the class (no state usage)

Reply:
Yeah, it’s a good utility function.

Reply:
- [x] move to utility function